### ISSUE TITLE
Web player doesn't restore aspect ratio

### ISSUE THREAD

**Issue by gudvinr** [jellyfin/jellyfin-web#3084]
### Describe the bug

1. Open video file
2. Change aspect ratio
3. Close app
4. Open app again
5. Chose same video file

Video playback starts. Subtitles that were chosen (#525 btw) are restored but aspect ratio isn't.

### Application version

2.3.3

### Where did you install the app from?

Google Play

### Device information

OnePlus 7 Pro

### Android version

Android 11 (Oxygen OS)

### Jellyfin server version

10.7.7

### Which video player implementations does this bug apply to?

- [X] Web player (default)
- [ ] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by gudvinr** [comment]
> If you're the original submitter of this issue, please comment confirming if this issue still affects you in the latest release or master branch, or close the issue if it has been fixed. 

Yes, issue still very much present. In fact, it is also the case for integrated player too

**Comment by thornbill** [comment]
I'm not sure what you mean by aspect ratio in this case. Could you provide screenshots? Thanks!

**Comment by gudvinr** [comment]
@thornbill I meant parameter in gear menu (right to volume bar) that is called "Aspect Ratio"
